python/ray/_private/usage/usage_lib.py

```diff
@@ -494,6 +492,18 @@ def set_usage_stats_enabled_via_env_var(enabled) -> None:
     os.environ[usage_constant.USAGE_STATS_ENABLED_ENV_VAR] = "1" if enabled else "0"
 
 
+def set_usage_stats_ray_init_cluster() -> None:
```

Comment:
this seems a bit hacky.. is there any way we just figure this out from existing path?

Reply:
Or alternatively, we just pass a flag when we start a dashboard instead?